numpy/core/tests/test_multiarray.py

```diff
@@ -342,6 +342,10 @@ def test_array_cont(self):
         assert_(np.ascontiguousarray(d).flags.c_contiguous)
         assert_(np.asfortranarray(d).flags.f_contiguous)
 
+    def test_npy_char_deprecation(self):
```

Comment:
well, I guess this would work out with that test, if you just return the descriptor in the test function (since it should be NULL on error I guess).